cli/command/container/cmd.go

```diff
@@ -12,9 +12,10 @@ import (
 // NewContainerCommand returns a cobra command for `container` subcommands
 func NewContainerCommand(dockerCli *command.DockerCli) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "container",
-		Short: "Manage containers",
-		Args:  cli.NoArgs,
+		Use:     "container",
+		Short:   "Manage containers",
+		Args:    cli.NoArgs,
+		Aliases: []string{"con"},
```

Comment:
`ctn` seems confusing, as would forget if it was `cnt` which is also a valid shortening, so I think I prefer the short `con`.